streams/authorizer/src/main/java/com/hortonworks/streamline/streams/security/Roles.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2017 Hortonworks.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+
+ *   http://www.apache.org/licenses/LICENSE-2.0
+
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+package com.hortonworks.streamline.streams.security;
+
+public class Roles {
+    public static final String ROLE_ADMIN = "ROLE_ADMIN";
+    public static final String ROLE_SECURITY_ADMIN = "ROLE_SECURITY_ADMIN";
```

Comment:
we should reduce the number of roles here. We can keep the "Admin", "Developer", "OP" and treat notifications and files as resources